lib/galaxy/managers/group_roles.py

```diff
@@ -59,8 +52,7 @@ def update(self, trans: ProvidesAppContext, id: EncodedDatabaseIdField, group_id
         group_role = self._get_group_role(trans, group, role)
         if group_role is None:
             self._add_role_to_group(trans, group, role)
-
-        return self._serialize_group_role(group_id, role)
+        return role
```

Comment:
```suggestion
        return group_role
```
Unless I am missing something?

Reply:
See https://github.com/galaxyproject/galaxy/pull/12337#discussion_r691277797